src/content/faq/General.md

```diff
@@ -3,7 +3,11 @@ title: Frequently Asked Questions (FAQ)
 layout: faq
 category: General
 permalink: /faq/
-questions: Why should I use GraphQL?,Is GraphQL a database language like SQL?,Does GraphQL replace REST?,How can I learn GraphQL?,Is GraphQL frontend or backend?,Is GraphQL only for React or JavaScript developers?,What is a GraphQL client and why would I use one?,Is GraphQL owned by Facebook?,What is the GraphQL Foundation?,How can I contribute to the GraphQL specification?
+gettingStartedQuestions: Why should I use GraphQL?,Does GraphQL replace REST?,How can I learn GraphQL?,Is GraphQL a database language like SQL?,Is GraphQL only for React or JavaScript developers?
+generalQuestions: Is GraphQL frontend or backend?,Does GraphQL use HTTP?,How does GraphQL affect my product’s performance?,What is a GraphQL client and why would I use one?,Does GraphQL replace ORMs?,Is GraphQL owned by Facebook?,Who is behind GraphQL?,What is the GraphQL Foundation?
+bestPracticesQuestions: Is GraphQL scalable?,Does GraphQL support offline usage?,What are the security concerns with GraphQL?,How can I set up authorization with GraphQL?,How does authentication work with GraphQL?,Is GraphQL the right fit for designing a microservice architecture?,How does versioning work in GraphQL?,How can I document my GraphQL API?
+specificationQuestions: What is the best way to follow specification releases?,How can I contribute to the GraphQL specification?,Where is the documentation for subscriptions?
+frontendQuestions: Does GraphQL replace Redux or other state management libraries?
```

Comment:
Idea: We can convert it to MDX and split out sections into separate MD files.
MDX files can include MD files: https://mdxjs.com/getting-started#documents

Reply:
They are all split into separate MD files in #985 - the design in this PR was a placeholder so people could review the content.

Reply:
Converting to MDX could def be an idea for the future though, I can open an issue about it once these PRs are merged!